### ISSUE TITLE
bigquery/tables code create table is broken

### ISSUE THREAD

**Issue by c0b** [GoogleCloudPlatform/nodejs-docs-samples#223]
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/bigquery/tables.js#L24

the createTable is defined with 4 arguments but called with only 3 arguments, caused runtime broken of callback not defined

``` js
function createTable (datasetId, tableId, schema, callback) {
```

https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/bigquery/tables.js#L233

``` js
var program = module.exports = {
  createTable: createTable,
  // ...
}

  .command('create <datasetId> <tableId>', 'Create a new table with the specified ID in the specified dataset.', {}, function (options) {
    program.createTable(options.datasetId, options.tableId, utils.makeHandler(false));
  })
```

**Comment by jmdobry** [comment]
Good catch! Sorry about that.

**Comment by jmdobry** [comment]
Fixed

**Comment by c0b** [comment]
why don't you put `Fixed #223` in your commit msg where you fixed it, then github would create connection between your commit to this issue; it's a general good practice because otherwise people need to spend time find out where or how you fix

**Comment by jmdobry** [comment]
I'm aware of this functionality, I had simply forgotten about this issue when I cleaned up the BigQuery samples. Here, I'll retroactively reference this issue:
